components/core/src/clp/regex_utils/RegexToWildcardTranslatorConfig.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef CLP_REGEX_UTILS_REGEXTOWILDCARDTRANSLATORCONFIG_HPP
+#define CLP_REGEX_UTILS_REGEXTOWILDCARDTRANSLATORCONFIG_HPP
+
+namespace clp::regex_utils {
+
+class RegexToWildcardTranslatorConfig {
+public:
+    // Constructors
+    RegexToWildcardTranslatorConfig(
+            bool case_insensitive_wildcard,
+            bool add_prefix_suffix_wildcards
+    )
+            : m_case_insensitive_wildcard{case_insensitive_wildcard},
+              m_add_prefix_suffix_wildcards{add_prefix_suffix_wildcards} {};
+
+    // Getters
+
+    /**
+     * @return True if the final translated wildcard string will be fed into
+     *         a case-insensitive wildcard analyzer. In such cases, we can
+     *         safely translate charset patterns such as [aA] [Bb] into singular
+     *         lowercase characters a, b.
```

Comment:
```suggestion
     * @return True if the final translated wildcard string will be fed into a case-insensitive
     * wildcard analyzer. In such cases, we can safely translate charset patterns such as [aA] [Bb]
     * into singular lowercase characters a, b.
```
1. The latest guideline changed to use 100 char for comments (80 char standard is deprecated)
2. We don't indent multi-line comments in C++